engine/src/flutter/shell/platform/embedder/embedder_semantics_update.cc

```diff
@@ -26,6 +27,114 @@ EmbedderSemanticsUpdate::EmbedderSemanticsUpdate(
   };
 }
 
+FlutterSemanticsFlag SemanticsFlagtoInt(const SemanticsFlags2& flags) {
```

Comment:
if this is only needed for jni, this method should probably be implement in jni instead of here.

Reply:
This is not for jni, 
in embedder.h I added a struct `flags2` but kept the old int `flags`.
this is used to set the value for the old int `flags`  in embedder.h. 

I actually don't know if setting the value for it is necessary 😅 , do you know the answer?

Reply:
we should make sure old semantics flag still reflects in the old int in embedder.h

Reply:
just for my curiosity, what will happen if we don't set the value for the  old semantics flag?